app/views/tube_creation/pooled_tubes_by_sample.html.erb

```diff
@@ -3,12 +3,6 @@
     <%= card title: 'Help' do %>
       <p>Upload the tube rack scan csv file describing the positions and barcodes of the tubes. An example is shown below:</p>
       <table class='table table-sm'>
-        <thead>
```

Comment:
Are the headers not needed anymore

Reply:
Yeah I'm not sure why, but the guidance was wrong - it actually will error if it has headers